connector.rb

```diff
@@ -15,26 +15,27 @@
         name: 'account',
         label: '4me Account',
         optional: false,
-        hint: 'The 4me account identifier'
+        hint: "You can find the 4me account identifier via 'Settings' => 'Account overview'."
       },
       {
         name: 'instance',
-        label: '4me Instance',
-        hint: 'The 4me instance',
+        label: '4me Environment',
+        hint: 'The environment of the 4me instance. ' \
+              '<a href="https://developer.4me.com/graphql/#service-url-1" target="_blank">Learn more</a>',
         optional: false,
         control_type: 'select',
         options: [
           ['Production', 'production'],
           ['Quality Assurance', 'quality_assurance'],
-          ['Demo', 'demo'],
-          ['Custom domain', 'custom_domain']
```

Comment:
Could you explain the motivation behind removing this field from the connection block. The reason I am asking this is because we will need to implement a workaround in the main repo to ensure backward compatibility for users who chose custom domain previously, but if it is already not supported anymore, we don't need to make this workaround.

Reply:
@klaasvandeweerdt , how do you know that only 4me used this custom_domain?

Reply:
@thawalk, the connector does not contains `secure_tunnel: true` which makes on-prem connection impossible, and in 4me you can only have a custom domain name when running an on-prem version of 4me.

Reply:
Got it, thanks!